lib/github.js

```diff
@@ -39,6 +39,8 @@ export default {
   createStatus: async (owner, repo, head_sha, entries) => {
     const github = await getRestClient();
 
+    const links = "### Download link(s):" + entries.map(entry => `- [${entry.artifactId}](${entry.url})`).join("\n")
```

Comment:
untested but something like this will pluralise, (it would be cleaner as a function)

```suggestion
    const links = `### Download link${entries.length > 1 ? 's' : ''}:` + entries.map(entry => `- [${entry.artifactId}](${entry.url})`).join("\n")
```